GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Node.js Web Team Governance
+
+The Node.js Web Team (@nodejs/web) is a team in the Node.js Project that is composed by a set of subteams. Each containing specific responsibilities and goals.
+
+### Node.js Website Team (`@nodejs/nodejs-website`)
+
+The Node.js Website Team is responsible for the day-to-day technical development of the Node.js Website. This is primarily the development of the website itself, adding new features, pages and components, but also fixing any security issues in the website code, handling operational maintenance, and so on.
+
+The maintainers on the Node.js Website Team are responsible for steering the technical direction of the Node.js Website, and reserve the right to make final decisions on any issues or pull requests, in line with the Contribution Guidelines, Collaborator Guidelines, the Code of Conduct and the overall Governance premises of the Node.js project.
+
+Members of this team are nominated through the guidelines provided in the Collaborator Guidelines within this repository.
+
+### Node.js Web Infra Team (`@nodejs/web-infra`)
+
+The Node.js Web Infra Team is responsible for maintaining the Infrastructure relating to Node.js's Web Presence. The Node.js Web Infra team has the responsibilities of:
+
+- Maintaining CI/CD pipelines related to Web Infrastructure
+- Maintaining our Infrastructure Providers (such as Vercel and Cloudflare, for the Node.js website)
```

Comment:
I'm not sure we have agreement on expanding maintenence of CloudFlare outside of the Build WG: https://github.com/nodejs/admin/issues/833
I'll bring up the issue in tomorrow's Build WG meeting https://github.com/nodejs/build/issues/3500

Reply:
Removed for now: https://github.com/nodejs/nodejs.org/pull/5779/commits/b2ff3f678a0f58ac0c721256df22b627708f906f